atlp-rwanda/hackers-ec-Fe
# #187419009 User product page

### **_PR decription_**
User product page, the product page UI should be different according to user roles.
- There is two users to view products, one is a seller and other one is buyer
- Buyers product page is under the hero section and seller's products are in seller's dashbord

### How to test this in local machine
- clone the repo and navigate to ```feat-product-list-187419009```
- Pull from origin
- run ```npm run dev```
-  check those pages 

Pivotal tracker id:

     ##187419009
